examples/_test_reconstruct.py

```diff
@@ -0,0 +1,181 @@
+import os
+import subprocess
+import typing
+
+import alifedata_phyloinformatics_convert as apc
+import pandas as pd
+
+from hstrat._auxiliary_lib import (
+    alifestd_collapse_unifurcations,
+    alifestd_count_leaf_nodes,
+    alifestd_prune_extinct_lineages_asexual,
+    alifestd_try_add_ancestor_list_col,
+)
+
+try:
+    import tqdist
+except (ImportError, ModuleNotFoundError) as e:
+    print("tqdist required for dstream surf evolution example")
+    print("python3 -m pip install tqdist")
+    raise e
+
+
+# adapted from https://github.com/mmore500/hstrat/blob/d23917cf03ba59061ff2f9b951efe79e995eb4d8/tests/test_hstrat/test_phylogenetic_inference/test_tree/_impl/_tree_quartet_distance.py
+def calc_triplet_distance(ref: pd.DataFrame, cmp: pd.DataFrame) -> float:
```

Comment:
This is useful enough we should probably go ahead and put a pandas implementation into `auxiliary_lib`

Reply:
(it's getting used in multiple locations/multiple projects)